### PR TITLE
CLF gets faster suicide vest detonate time

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request
cut down from 2 seconds to 0.5
## Why It's Good For The Game
current vest detonate time takes too long for hvh gameplay and how weak clf are
## Changelog
:cl:
balance: clf suicide vest detonates faster
/:cl:

### CODE REVIEW

**Review by TiviPlus — APPROVED** [review]
Code ok

**Comment by Wisemonster** [comment]
Doesn't the vest have deva on it? I don't think it's a good idea to have an insta-deva vest, considering that means they can instantly delete things like tanky marines, corpman/FCs, and mechs. Maybe making it provide damage resistance instead would be better, considering you would still have 2 seconds to deal with the suicide vester, while still making it more useful.

**Review by Kuro020 — APPROVED** [review]

File: code/game/objects/items/explosives/bombvest.dm [hunk 1]
```diff
@@ -112,3 +115,6 @@
 		appendage.droplimb()
 	explosion(target, 15, 0, 0, 0, 15, 15)
 	qdel(src)
+
+/obj/item/clothing/suit/storage/marine/boomvest/fast
+	detonate_time = 5
```
Comment (Lumipharon) [inline code/game/objects/items/explosives/bombvest.dm:120]:
```suggestion
	detonate_time = 0.5 SECONDS
```

**Comment by Ragecrabbo** [comment]
> current vest detonate time takes too long for hvh gameplay

Man it takes too long in general but somehow the gamemode where it has the potential to actually do something (instagibs marines, tanks and mechs) is where its getting a buff.

**Pull request merged**